actionmailbox/.gitignore

```diff
@@ -1,3 +1,5 @@
+/test/dummy/storage/*.sqlite3
+/test/dummy/storage/*.sqlite3-*
 /test/dummy/db/*.sqlite3
 /test/dummy/db/*.sqlite3-*
```

Comment:
Do we need to remove these two lines above?

Reply:
As I wrote in the PR body,

> I intentionally left the old db/ entries as is, for those who already have such files in their disk.

is my intention. Not sure if this applies to everyone though.

Reply:
Sorry, I missed your comment.

I think these two lines should be removed because these .gitignore files are for Rails contributors, not for Rails users. Under the current main branch, all the sqlite3 database files are always stored in the `storage` directory.

Reply:
> because these .gitignore files are for Rails contributors, not for Rails users. Under the current main branch, all the sqlite3 database files are always stored in the `storage` directory.

I of course know this, and what I mean is that removing this now would unnecessarily introduce a new risk of committing .sqlite3 files that are created in the past, whereas there seems no downside of leaving these old configurations for a while.

Reply:
Understood.

Reply:
Thanks. Actually I'm sure you will find some if you try this on your laptop 😉 
`$ find . -name '*.sqlite3'`